util/testkit/testkit.go

```diff
@@ -170,6 +170,12 @@ func (tk *TestKit) MustQuery(sql string, args ...interface{}) *Result {
 	rs, err := tk.Exec(sql, args...)
 	tk.c.Assert(errors.ErrorStack(err), check.Equals, "", comment)
 	tk.c.Assert(rs, check.NotNil, comment)
+	return tk.ResultsToStr(rs, comment)
+}
+
+// ResultsToStr converts ast.RecordSet to testkit.Result.
+// It is used to check results of execute statement in binary mode.
+func (tk *TestKit) ResultsToStr(rs ast.RecordSet, comment check.CommentInterface) *Result {
```

Comment:
How about name it `RecordSetToResult`, the return value is not a string...

Reply:
It is basically a string array with comment. Anyway I'll change the name.